Documentation/user-guides/scrapeconfig.md

```diff
@@ -0,0 +1,137 @@
+---
+weight: 153
+toc: true
+title: ScrapeConfig CRD
+menu:
+    docs:
+        parent: user-guides
+lead: ""
+images: []
+draft: false
+description: Guide to use ScrapeConfig to scrape targets external to the Kubernetes cluster
+---
+
+Starting with prometheus-operator v0.65.x, one can use the `ScrapeConfig` CRD to scrape targets external to the
+Kubernetes cluster or create scrape configurations that are not possible with the higher level
+`ServiceMonitor`/`Probe`/`PodMonitor` resources.
+
+# Prerequisites
+* `prometheus-operator` `>v0.65.1`
+* `ScrapeConfig` installed in the cluster. Make sure to restart the operator after having pushed the CRD.
+
+## Configure Prometheus or PrometheusAgent to select ScrapeConfigs
```

Comment:
(nit) 
```suggestion
# Configure Prometheus or PrometheusAgent to select ScrapeConfigs
```

Reply:
This was for me part of the prerequisites but I'm not too fussed either way